scripts/scr_marine_struct/scr_marine_struct.gml

```diff
@@ -523,148 +523,190 @@ global.base_stats = { //tempory stats subject to change by anyone that wishes to
+			start_gear:{"armour":"Skitarii Armour", "wep1":"Hellgun", "wep2":"Combat Knife"},
 			base_group : "skitarii",
 	},
 	"tech_priest":{
-			strength:[12,1],
-			constitution:[30,1],
-			dexterity:[20,1],
-			weapon_skill : [15,5],
-			ballistic_skill : [15,5],				
-			intelligence:[30,3],
-			wisdom:[20,3],
-			charisma :[8,1],
-			religion : "cult_mechanicus",
 			title : "Tech Priest",
+			strength : [15,1],
+			constitution : [30,1],
+			weapon_skill : [15,1],
+			ballistic_skill : [25,3],				
+			dexterity : [25,3],
+			intelligence : [30,3],
+			wisdom : [20,2],
+			charisma : [30,2], // Considering their voicelines in DoW:DC and SS, I'd say they can have charisma
+			religion : "cult_mechanicus",
 			piety : [45,3],
-			luck :6,
-			technology :[55,3],
-			skills: {weapons:{"power_weapon":2,}},	
-			start_gear:{"armour":"dragon_scales", "wep1":"power_weapon"},
+			luck : 10,
+			technology : [55,3],
+			skills: {weapons:{"power_weapon":2,}}, // Does this even work?
+			start_gear:{"armour":"dragon_scales", "wep1":"power_weapon"}, // Gear items are not mentioned here
 			base_group : "tech_priest",
 	},
-	"skitarii_ranger":{
-			title : "Skitarii Ranger",
-			strength:20,
-			constitution:26,
-			weapon_skill : [20,5],
-			ballistic_skill : [20,5],				
-			dexterity:34,
-			intelligence:7,
-			wisdom:2,
-			charisma :2,
-			religion : "cult_mechanicus",
-			piety : 10,
-			technology :8,
-			luck :5,
-			weapon_skill : 35,
-			ballistic_skill : 40,			
-			skills: {weapons:{"ranger_long_rifle":1,}},	
-			start_gear:{"armour":"skitarii_armour", "wep1":"hellgun", "wep2":"shuriken_pistol"},
-			base_group : "skitarii",
+	"skitarii_ranger":{ // TODO rename this
+			title : "Skitarii Ranger", // TODO - that should be Eldar
+			strength : [25,1],
+			constitution : [30,2],
+			weapon_skill : [45,4],
+			ballistic_skill : [45,4],				
+			dexterity : [50,5], // Dexterity should be eldar 'specialization'
+			intelligence : [35,3],
+			wisdom : [50,3],
+			charisma : [20,2], // Arrogance from cultural stuff, supposedly
+			religion : "cult_mechanicus", // TODO - add eldar faith
+			piety : [30,5], // I think eldar rangers can be either - faithful to the path or more cynical
+			luck : 10,
+			technology : [20,1], // Elves in most fiction tend to be kind of bad at technology, right?
+			skills: {
+				weapons:{
+					"Ranger Long Rifle":3, "Shuriken Pistol":3, "Eldar Power Sword":3}}, // TODO - check if these actually work
+			start_gear:{"armour":"Power Armour", "wep1":"Ranger Long Rifle", "wep2":"Eldar Power Sword"}, // TODO - add Eldar Armour
+			base_group : "skitarii", // Might want to rename this
 	},
 	"inquisition_crusader":{
-			title : "Inquisition Crusador",
-			strength:4,
-			constitution:26,
-			dexterity:4,
-			intelligence:3,
-			wisdom:2,
-			charisma :2,
+			title : "Inquisition Crusader",
+			strength : [10,1], // 10 is considered standard for a chad-like Imperial Guardsman
+			constitution : [10,1],
```

Comment:
And you've nerfed crusader hp.

Reply:
Yeah, since they are regular humans, supposedly.
Or, are we going to treat them as something else? We probably need stat-buffing (constitution mostly) equipment.